pkgs/by-name/ze/zenmap/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchzip,
+  nmap,
+  python3Packages,
+  wrapGAppsHook3,
+  gobject-introspection,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "zenmap";
+  version = "7.95";
+  pyproject = true;
+
+  src = fetchzip {
+    url = "https://nmap.org/dist/nmap-${version}.tar.bz2";
+    hash = "sha256-sikDuQb0uI+/Ryy0Evt67LHw7GzTIX8xFy/AyHxEUsg=";
+  };
+
+  sourceRoot = "${src.name}/zenmap";
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+  ];
```

Comment:
may could just go in deps? not sure.

Reply:
If it's a library we link against and need at runtime (not only build-time) it should go into `buildInputs`.